composer.json

```diff
@@ -19,8 +19,8 @@
     "require": {
         "php": ">=8.1",
         "behat/behat": "^3",
-        "behat/mink": "<=1.10",
-        "behat/mink-selenium2-driver": "<=1.6",
+        "behat/mink": "<=1.11",
```

Comment:
this constraint should be now saying `>=1.11` and `>=1.7`

Reply:
Hi @AlexSkrypnyk 
I have update constraint as your requested.

I have re-reviewed the changes and handling of keys in our behat-step then could not find anything wrong.
Our behat-step using `withSyn` and `executeJsOnXpath` method on mink selenium, only `withSyn` method change as `Syn` to `syn`.

But I found other thing is the our keyboard trait look like does not handle correct even in main branch. 
Please refer https://github.com/drevops/behat-steps/pull/189